addons/product/views/product_document_views.xml

```diff
@@ -56,51 +56,34 @@
                 <templates>
                     <t t-name="kanban-menu">
-                        <a t-if="widget.editable" type="edit" class="dropdown-item">Edit</a>
+                        <a t-if="widget.editable" type="open" class="dropdown-item">Edit</a>
                         <a t-if="widget.deletable" type="delete" class="dropdown-item">Delete</a>
                         <a t-attf-href="/web/content/#{record.ir_attachment_id.raw_value}?download=true" download="" class="dropdown-item">Download</a>
                     </t>
-                    <t t-name="kanban-box">
-                        <div class="oe_kanban_global_area o_kanban_attachment">
-                            <field name="res_model" invisible="True"/>
-                            <div class="ribbon ribbon-top-right" invisible="active">
-                                <span class="text-bg-danger">Archived</span>
+                    <t t-name="kanban-card" class="o_kanban_attachment flex-row">
+                        <widget name="web_ribbon" title="Archived" bg_color="text-bg-danger" invisible="active"/>
+                        <widget name="web_ribbon" title="Variant" bg_color="text-bg-secondary" invisible="not active or res_model != 'product.product'" groups="product.group_product_variant"/>
+                        <aside class="o_kanban_image m-2">
+                            <t t-set="webimage" t-value="new RegExp('image.*(gif|jpeg|jpg|png|webp)').test(record.mimetype.value)"/>
+                            <t t-set="binaryPreviewable"
+                                t-value="new RegExp('(image|video|application/pdf|text)').test(record.mimetype.value) &amp;&amp; record.type.raw_value === 'binary'"/>
+                            <div t-attf-class="o_kanban_image_wrapper #{(webimage or binaryPreviewable) ? 'o_kanban_previewer' : ''}">
+                                <div t-if="record.type.raw_value == 'url'" class="o_url_image fa fa-link fa-3x text-muted" aria-label="Image is a link"/>
+                                <img t-elif="webimage" t-attf-src="/web/image/#{record.ir_attachment_id.raw_value}" width="100" height="100" alt="Document" class="o_attachment_image"/>
+                                <div t-else="" class="o_image o_image_thumbnail" t-att-data-mimetype="record.mimetype.value"/>
                             </div>
-                            <div class="ribbon ribbon-top-right" invisible="not active or res_model != 'product.product'" groups="product.group_product_variant">
-                                <span class="text-bg-secondary">Variant</span>
+                        </aside>
+                        <main class="ms-2 p-2">
+                            <field name="name" class=" fw-bolder fs-5 text-truncate"/>
+                            <field  t-if="record.type.raw_value == 'url'" name="url" widget="url" invisible="type == 'binary'"/>
+                            <div class="d-flex flex-column mt-2">
```

Comment:
is this div still necessary?
If not xpaths in other views need to be rewritten

Reply:
yes we need this, in enterprise [PR](https://github.com/odoo/enterprise/pull/67076/files#diff-bfd35f78ab40c0d280c0bc457136d58dc811c47a35f08032f27529bedf2f87d6R25) you can see that this whole div is getting replaced and in with all other xpath's in community we are add thing `inside` it, so if we remove the div then only one field can replace other fields will stay in the view.